server/responses/types.ts

```diff
@@ -43,3 +43,27 @@ export interface AdventureResponse {
 export type AdventuresResponse = {
   adventures: AdventureResponse[];
 }
+
+export interface InitiativeOrderItemResponse {
+  entityId: string;
+  entityType: 'creature' | 'player';
+  id: string;
+  imageSrc: string;
+  name: string;
+  resourceA:  number;
+  resourceB: number;
+  sortValue: number;
+  visibilityState: 'on' | 'hidden' | 'removed';
```

Comment:
Same thing for visibilityState, maybe these can be enums?

Reply:
Absolutely they should